src/components/views/BlackCardSelection.js

```diff
@@ -6,6 +6,20 @@ import {useHistory, useLocation} from "react-router-dom";
 import BaseContainer from "../ui/BaseContainer";
 import PropTypes from "prop-types";
 
+function msToHM( ms ) {
+    // convert milliseconds to seonds:
+    let secs = ms / 1000;
+    // get hours
+    const hours = parseInt( secs/3600);
```

Comment:
I would not use parseInt here, because parseInt is used for parsing a string. Better use Math.floor() for integer division.

Reply:
changed the hole function anyway